Proposal-eng.md

```diff
@@ -0,0 +1,64 @@
+# Proposal - UIC HCC Computer Society
+
+Dear Sir / Madam:
+
+
+
+We are computer technology enthusiasts in UIC from different majors and different grades, who founded UIC HCC Computer Society (Unofficial) in October, 2016, in hope of spreading the knowledge of computer science and technology, conveying the sprit of open source, and sharing as well as focusing on the frontier development of computer science. Today, we wish to establish UIC HCC Computer Society officially under Innovation Center at the moment when UIC New Campus has moved smoothly and Innovation Center will be established soon. We hope to make contributions to students and staffs in UIC as well as the Industry-University Integration plan of UIC with the support of this platform.
+
+This proposal will introduce overview, vision, mission, task and expected achievement of UIC HCC Computer Society (hereafter this text will be abbreviated as Society). In addition, we will list our expectations to Innovation Center and UIC.
+
+
+
+## Society Overview
+
+UIC HCC Computer Society established unofficially in October, 2016, and have 20 members from different majors and different grades (members list in appendix). We make our target **Bridging You And Technology** as our responsibility, and spread knowledge related to computer science to UIC teachers and students in different occasions actively. With the emerge of new technologies in computer science, we find it difficult to achieve our target and keep our development under the unofficial operation mode. Therefore, our society wish to obtain the strong support from UIC and Innovation Center to carry out more science literacy, practice and research activities.
```

Comment:
1. maybe replace the word target with goal would be better
2. the last sentence, suggest express like "carry out more science-popularising lectures, practices, and research activities"

Reply:
1. Fixed.
2. Will fix.